examples/run-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
Where you planning to wire this to Travis too maybe?

Reply:
Yes, but not right away as I'm not sure this branch has all the examples working.